tests/cases/project/declarationsExportDefault.json

```diff
@@ -0,0 +1,17 @@
+{
+    "scenario": "declarations_ExportDefault",
+    "projectRoot": "tests/cases/projects/declarations_ExportDefault",
+    "declaration": true,
+    "baselineCheck": true,
+    "inputFiles": [
+        "constEnum.ts",
+        "declareClass.ts",
+        "declareConstEnum.ts",
+        "declareEnum.ts",
+        "declareFunction.ts",
+        "declareNamespace.ts",
+        "enum.ts",
+        "namespace.ts",
+        "type.ts"
+    ]
+}
```

Comment:
I added this project test to test the .d.ts output, is there a better way?

Reply:
We are trying to avoid adding new "projects" tests in favor test variations in compiler/conformance tests. You can set `// @declaration: true` in the compiler/conformance tests to get .d.ts output.